apps/back-office/src/app/(dashboard)/settings/team-members/add/page.tsx

```
@@ -0,0 +1,221 @@
+      type: "",
+      number: "",
+      expirationDate: "",
+      state: "",
+    },
+  });
+
+  // This effect runs when form inputs change
+  useEffect(() => {
+    const handleFormInputChange = (event: Event) => {
+      const input = event.target as HTMLInputElement;
+      if (input && input.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)